util/src/main/java/org/datacommons/util/McfChecker.java

```diff
@@ -54,17 +54,25 @@ public class McfChecker {
   private ExistenceChecker existenceChecker;
   private StatVarState svState;
   boolean shouldCheckObservationAbout; // Should check for observationAbout existence on SVObs
+  boolean allowNanSVObs;
 
   // Argument |graph| may be Instance or Template MCF.
   public static boolean check(
       Mcf.McfGraph graph,
       ExistenceChecker existenceChecker,
       StatVarState svState,
       boolean shouldCheckObservationAbout,
+      boolean allowNanSVObs,
       LogWrapper logCtx)
       throws IOException, InterruptedException {
     return new McfChecker(
-            graph, null, existenceChecker, svState, shouldCheckObservationAbout, logCtx)
+            graph,
+            null,
+            existenceChecker,
```

Comment:
not related to this PR:  It might be worth putting the args to a "struct" like class and passing that into various util functions here and elsewhere.  add a TODO?